app/views/home/index.html.erb

```diff
@@ -1,2 +1 @@
-<h1>Home#index</h1>
-<p>Find me in app/views/home/index.html.erb</p>
+redirect_to "/tasks"
```

Comment:
Since `redirect_to` is a Ruby method, this will only display the text `redirect_to "/tasks"`, it won't actually *do* the redirect and send the user somewhere else. It looks like you already this in the controller for this view template, so it should be safe to remove this file if you want.